client/packages/system/src/Item/Components/StockItemSearchInput/StockItemSearchInput.tsx

```diff
@@ -50,11 +64,17 @@ export const StockItemSearchInput: FC<StockItemSearchInputProps> = ({
   const formatNumber = useFormatNumber();
   const selectControl = useToggle();
 
-  const debounceOnFilter = useDebounceCallback(
-    (searchText: string) => onFilter(searchText),
-    [onFilter],
-    DEBOUNCE_TIMEOUT
-  );
+  const filterByNameAndCode = (
+    options: ItemStockOnHandFragment[],
+    state: FilterOptionsState<ItemStockOnHandFragment>
+  ) =>
+    options.filter(
+      option =>
+        option.code.includes(state.inputValue) ||
+        option.name.includes(
+          getItemNameFilterValue(state.inputValue, selectedCode)
+        )
+    );
```

Comment:
This was needed to have the `{code} {name}` display, but show results when _either_ code or name matches